backend/inspirehep/disambiguation/tasks.py

```diff
@@ -196,6 +173,30 @@ def create_new_author(full_name, from_recid):
     return new_author
 
 
+def match_author(author):
+    matched_authors = match(author, current_app.config["AUTHOR_MATCHER_EXACT_CONFIG"])
```

Comment:
are you avoiding the match based on existing BAI in the record? I don't see it.

Reply:
Yes, I've added it to the validator.

Reply:
:+1: I missed it